virtool/administrators/api.py

```diff
@@ -33,6 +34,7 @@
 
 
 @routes.view("/admin/roles")
+@flag(FlagName.FF_ADMINISTRATOR_ROLES.value)
```

Comment:
Should look like this. See other comments for explanation.
```suggestion
@flag(FlagName.ADMINISTRATOR_ROLES)
```